src/PostReportController.php

```diff
@@ -5,10 +5,12 @@
  * @package P4MT
  */
 
+namespace P4\MasterTheme;
+
 /**
- * Class P4_Post_Report_Controller
+ * Class P4\MasterTheme\P4_Post_Report_Controller
  */
-class P4_Post_Report_Controller {
+class PostReportController {
```

Comment:
Can we make the classname to `PostReport` only?

Reply:
We would need to check if something still uses the old class name, as currently the fallback transforms the old class name into the new one to find the file. But maybe let's do renames separately after this is merged? There are a couple more that could use more accurate naming (e.g. `ImageCompression` which does more than compression), so could make sense to look at these together, as we might also introduce some folders.

Reply:
But definitely well spotted that we should change this :100: Controller is a term that is used for so many things that it doesn't add any meaning but rather adds confusion.